### DISCUSSION TITLE
clean .class file generated

### DISCUSSION THREAD

**Discussion by HzEdward** [redhat-developer/vscode-java#3530] [Q&A]
I run the java programme directly by clicking on "run" button the codeLens, and from the console I could tell that the .class file is created in: redhat.java/jdt_ws/xxxxxxx/bin. 
So, 1. would this class file be automatically cleaned by the extension?
2. Is there any configuration option I could do to put the class file in my working directory?

**Comment by ForsakenDelusion** [comment]
i have the same question,but,you can create a  java project,so that you can  see the setting.json in your wordspace,.
